src/EPR.Calculator.Frontend/appsettings.Development.json

```diff
@@ -1,11 +1,28 @@
 {
+  "AzureAd": {
```

Comment:
Just to make sure did you added/requested to add these configs in epr-app-config-settings?

Reply:
All done. Thanks